tests/integration/components/frost-text-test.js

```diff
@@ -16,29 +19,301 @@ describeComponent(
     })
 
     it('renders', function () {
-      this.render(hbs`{{frost-text}}`)
-      expect(this.$()).to.have.length(1)
+      this.render(hbs`
+          {{frost-text}}
+      `)
+
+      expect(
+        this.$('input').attr('tabindex'),
+        'tabindex is set'
+      ).to.eql('0')
+
+      expect(
+        this.$('input').attr('type'),
```

Comment:
Everything worked except "autocorrect" after I changed .attr() to .prop(). 
I am getting this: AssertionError: autocorrect attribute is set: expected undefined to deeply equal 'on'
Did some research and it says it needs apple extension in order to work in the Safari HTML Reference. 
Please see the autocorrect section: 
https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocorrect